e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -492,55 +492,57 @@ describeEE("scenarios > embedding > full app", () => {
           },
         });
       });
-      cy.get("@postMessage")
-        .should("have.been.calledWith", {
+
+      // TODO: Find a way to assert that this is the last call.
+      cy.get("@postMessage").should(
+        "have.been.calledWith",
+        {
           metabase: {
             type: "frame",
             frame: {
               mode: "fit",
               height: Cypress.sinon.match(value => value > 1000),
             },
           },
-        })
-        .and("not.have.been.calledWith", {
+        },
+        "*",
```

Comment:
This has always been called, it was just never asserted explicitly. I changed this to make all assertions stay consistent. Since the last assertion always failed without `"*"` for some reason.